src/utils/storybook-utils.ts

```diff
@@ -0,0 +1,11 @@
+import { HashEntry } from "@storybook/manager-api";
+
+export function getStoryId(resolvedEntry: HashEntry): string {
+  if (!resolvedEntry) {
+    return null;
+  }
+  if (resolvedEntry.type === "component") {
+    return resolvedEntry.children[0];
+  }
```

Comment:
Can you elaborate here pls?

Reply:
When you have a story with the name Playground for example, or Button, it is the ComponentEntry.
Every component entry has one or more children (or none if nothing is rendered).
If you call your story Playground then the storyId is playground--playground, and I cannot count on all consumers to follow the same naming, so I need to resolve, according to their component entry, what's the story id, to allow them choose whichever name they decide on

Reply:
Too bad there isn't a better way to identify it in SB